e2e-tests/cypress/e2e/journeys/insurance/feedback/validation/feedback-validation.spec.js

```diff
@@ -0,0 +1,69 @@
+import partials from '../../../../partials';
+import { feedbackPage } from '../../../../pages/insurance/feedback';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import { FIELD_IDS } from '../../../../../../constants/field-ids';
+import { submitButton } from '../../../../pages/shared';
+
+const {
+  FEEDBACK: {
+    IMPROVEMENT,
+    OTHER_COMMENTS,
+  },
+} = FIELD_IDS;
+
+const {
+  START,
+  FEEDBACK,
+} = ROUTES.INSURANCE;
+
+const ERROR_MESSAGE_IMPROVEMENT = ERROR_MESSAGES[IMPROVEMENT];
+const ERROR_MESSAGE_OTHER_COMMENT = ERROR_MESSAGES[OTHER_COMMENTS];
+
+context('Insurance - Feedback - form validation', () => {
+  const startUrl = START;
+  const url = FEEDBACK;
+
+  before(() => {
+    cy.navigateToUrl(startUrl);
+    partials.phaseBanner.feedbackLink().click();
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  describe('when validation errors are present', () => {
```

Comment:
I think we can remove this line doesn't seem needed

Reply:
I thought to use it but it will double the duration of the test because we are testing 2 errors at the same time